core/modules/wsched/ChunkDisk.cc

```diff
@@ -101,13 +101,10 @@ void ChunkDisk::enqueue(wbase::Task::Ptr const& a) {
         _activeTasks.push(a);
         state = "EMPTY";
     } else {
-        if(chunkId < _chunkState.lastScan()) {
+        // To keep from getting stuck  on this chunkId, put new requests for this chunk on pending.
+        if(chunkId <= _chunkState.lastScan()) {
```

Comment:
Some behavior change here.  Suggest pulling these behavior changes, comment additions/cleanup, and unit tests into separate commits before merging?